CesiumGeospatial/src/BoundingRegion.cpp

```diff
@@ -223,7 +224,8 @@ BoundingRegion::computeUnion(const BoundingRegion& other) const noexcept {
   return BoundingRegion(
       this->_rectangle.computeUnion(other._rectangle),
       glm::min(this->_minimumHeight, other._minimumHeight),
-      glm::max(this->_maximumHeight, other._maximumHeight));
+      glm::max(this->_maximumHeight, other._maximumHeight),
+      this->_ellipsoid);
```

Comment:
Add the ellipsoid as a parameter to this function rather than using the field, so that we can eliminate the field.